2019-05-Berlin/agenda.md

```diff
@@ -0,0 +1,86 @@
+# Collaborator Summit Berlin 2019 - May 30th and 31th 🇩🇪
+
+This is a live document that will be constantly updated to match the proposed agenda.
+
+Our time will be best served with some monotrack presentations and discussions in the first day, and then breakout rooms in the second day. __We can change everything__, so if you think a better structure will suit us better, let us know in #135!
```

Comment:
Based on some of the feedbacks from Vancouver, we are best using our time if the sessions have quorum:

```suggestion
A collaborator summit is a mix of presentations and interactive sessions lead by the Teams, Working Groups, and Projects that participate in the summit. For interactive sessions, our time is best served if a the team/working groups/project participate in good numbers (quorum etc).

__We can change everything__, so if you think a better structure will suit us better, let us know in #135, or just send a PR!
```

Reply:
The specificity on which "entities" can lead the sessions is a solid improvement on the earlier language 👍🏽

Totally! Setting a level for quorum ahead of time is a clear way to find out whether the session will be of value. Can the proposals feasibly estimate the minimum numbers for their session to activate? If so, we can ask them to.

@joyeecheung did a rad survey of which sessions people would attend. This time it would be great to more deeply integrate that into the numbers planning!